docs/docs/how-tos/index.md

```diff
@@ -48,12 +48,24 @@ LangGraph makes it easy to manage conversation [memory](../concepts/memory.md) i
 [Human-in-the-loop](../concepts/human_in_the_loop.md) functionality allows
 you to involve humans in the decision-making process of your graph. These how-to guides show how to implement human-in-the-loop workflows in your graph.
 
-- [How to add breakpoints](human_in_the_loop/breakpoints.ipynb)
-- [How to add dynamic breakpoints](human_in_the_loop/dynamic_breakpoints.ipynb)
-- [How to edit graph state](human_in_the_loop/edit-graph-state.ipynb)
-- [How to wait for user input](human_in_the_loop/wait-user-input.ipynb)
+
+Key workflows:
+
+- [How to wait for user input](human_in_the_loop/wait-user-input.ipynb): A basic example that shows how to implement a human-in-the-loop workflow in your graph using the `interrupt` function.
+- [How to review tool calls](human_in_the_loop/review-tool-calls.ipynb): Incorporate human-in-the-loop for reviewing/editing/accepting tool call requests before they executed using the `interrupt` function.
+ 
+
```

Comment:
should we add `interrupt()` howto?

Reply:
No, I don't think so I removed it instead. I think we should avoid documenting how to use a particular part of the API in a how-to. How-tos are not API references, but a way to achieve specific objectives. The API reference for `interrupt()` has a basic usage example.

The easiest way to see that we have a problem is by looking at looking at the nonsensical content that we have for the how-to for HIL (e.g., how does updating state have anything to do with HIL? -- it's super unclear from looking at the index)